src/EntityFramework/Core/Mapping/MappingItemLoader.cs

```diff
@@ -2092,7 +2090,7 @@ private void ValidateFunctionImportMappingParameters(XPathNavigator nav, EdmFunc
             var lineInfo = new LineInfo(nav);
 
             // Process entity type.
-            var entityTypeString = GetAttributeValue(nav.Clone(), MslConstructs.EntitySetMappingTypeNameAttribute);
+            GetAttributeValue(nav.Clone(), MslConstructs.EntitySetMappingTypeNameAttribute);
```

Comment:
@lajones @divega I don't think there is an expert here, which is why I don't think we should be making any of these changes.  AFAIK, the only reason we are doing this is because of FxCop, and as far as I know the main reason FxCop is now flagging stuff is because we are running on debug builds, which is something we never did before because it generates too many false hits. I think it would be good to discuss in person why we are doing all this work.